core/admin/mailu/ui/templates/user/list.html

```diff
@@ -39,14 +39,14 @@
       <a href="{{ url_for('.fetch_list', user_email=user.email) }}" title="{% trans %}Fetched accounts{% endtrans %}"><i class="fa fa-download"></i></a>&nbsp;
     </td>
     <td>{{ user }}</td>
-    <td>
+    <td data-ordered="{{ user.enable_imap*2 + user.enable_pop }}">
       {% if user.enable_imap %}<span class="badge bg-info">imap</span>{% endif %}
       {% if user.enable_pop %}<span class="badge bg-info">pop3</span>{% endif %}
     </td>
-    <td>{{ user.quota_bytes_used | filesizeformat }} / {{ (user.quota_bytes | filesizeformat) if user.quota_bytes else '∞' }}</td>
+    <td data-ordered="{{ user.quota_bytes_used }}">{{ user.quota_bytes_used | filesizeformat }} / {{ (user.quota_bytes | filesizeformat) if user.quota_bytes else '∞' }}</td>
     <td>{{ user.comment or '-' }}</td>
-    <td>{{ user.created_at | format_date }}</td>
-    <td>{{ user.updated_at | format_date }}</td>
+    <td data-ordered="{{ user.created_at or '0000-00-00' }}">{{ user.created_at | format_date }}</td>
```

Comment:
When testing this PR, the sorting on this column works opposite as expected.
On the User List screen (https://hostname/admin/user/list/<domain>), I click on the Created column.

Ascending (Upward pointing arrow), the row with oldest date is on the top.
Descending (Downward pointing arrow), the row with newest date is on the top.

The Last Edit column works as expected.
![image](https://user-images.githubusercontent.com/52963853/200587187-4a63d501-407f-4087-ac42-cffb381c84b0.png)

Reply:
Is it working now?

Reply:
Yes it is!